### DISCUSSION TITLE
Attendees link

### DISCUSSION THREAD

**Discussion by carlbannister** [liedekef/events-made-easy#332] [Q&A]
Hi,

I currently use the #_EVENTPRINTBOOKINGSLINK. Is there a similar link to print Attendees?

Thanks

Carl

**Comment by liedekef** [comment]
Not really, but this is an alternative:
https://www.e-dynamics.be/wordpress/eme_attendees_report_link/

**Comment by carlbannister** [comment]
OK thanks, I did try the report link but didn't want to mess around with a
csv as it needs formating to view it properly.

Just need to make it simple for a quick view
